Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -1600,6 +1605,322 @@ def check_font_version_fields(fb, ttFont):
     fb.ok("All font version fields look good.")
 
 
+@register_test
+@old_style_test(
+    id='com.google.fonts/test/045'
+)
+def check_Digital_Signature_exists(fb, ttFont):
+  """Does the font have a DSIG table ?"""
+  if "DSIG" in ttFont:
+    fb.ok("Digital Signature (DSIG) exists.")
+  else:
+    fb.error("This font lacks a digital signature (DSIG table)."
+             " Some applications may require one (even if only a"
+             " dummy placeholder) in order to work properly.")
+
+
+@register_test
+@old_style_test(
+    id='com.google.fonts/test/046'
+)
+def check_font_contains_the_first_few_mandatory_glyphs(fb, ttFont):
+  """Font contains the first few mandatory glyphs
+     (.null or NULL, CR and space)?"""
+  # It would be good to also check
+  # for .notdef (codepoint = unspecified)
+  null = getGlyph(ttFont, 0x0000)
+  CR = getGlyph(ttFont, 0x000D)
+  space = getGlyph(ttFont, 0x0020)
+
+  missing = []
+  if null is None: missing.append("0x0000")
+  if CR is None: missing.append("0x000D")
+  if space is None: missing.append("0x0020")
+  if missing != []:
+    fb.warning(("Font is missing glyphs for"
+                " the following mandatory codepoints:"
+                " {}.").format(", ".join(missing)))
+  else:
+    fb.ok("Font contains the first few mandatory glyphs"
+          " (.null or NULL, CR and space).")
+
+
+@register_condition
+@condition
+def missing_whitespace_chars(ttFont):
+  space = getGlyph(ttFont, 0x0020)
+  nbsp = getGlyph(ttFont, 0x00A0)
+  # tab = getGlyph(ttFont, 0x0009)
+
+  missing = []
+  if space is None: missing.append("0x0020")
+  if nbsp is None: missing.append("0x00A0")
+  # fonts probably don't need an actual tab char
+  # if tab is None: missing.append("0x0009")
+  return missing
+
+
+@register_test
+@old_style_test(
+    id='com.google.fonts/test/047'
+  , conditions=['missing_whitespace_chars']
+)
+def check_font_contains_glyphs_for_whitespace_chars(fb,
+                                                    ttFont,
+                                                    missing_whitespace_chars):
+  """Font contains glyphs for whitespace characters?"""
+  if missing_whitespace_chars != []:
+    fb.error(("Whitespace glyphs missing for"
+              " the following codepoints:"
+              " {}.").format(", ".join(missing_whitespace_chars)))
+  else:
+    fb.ok("Font contains glyphs for whitespace characters.")
+
+
+@register_test
+@old_style_test(
+    id='com.google.fonts/test/048'
+  , conditions=['missing_whitespace_chars']
+)
+def check_font_has_proper_whitespace_glyph_names(fb,
+                                                 ttFont,
+                                                 missing_whitespace_chars):
+  """Font has **proper** whitespace glyph names?"""
+  if missing_whitespace_chars != []:
```

Comment:
Is the condition understood as "falsy" if its value is an empty list ?

In some places we need to explicitely skip because we want to provide a feedback message to the user explaining why it was skipped. Alternatively we could perhaps declare in the condition code what message to be displayed whenever the condition is false.